doc/Analytics.xml

```diff
@@ -3911,19 +3911,21 @@
     <title>Motion detection (normative)</title>
     <section>
       <title>Motion region detector</title>
-      <para>The region motion detector detects any motion against the specified motion region. The rule is configured for an area (region of interest on the image source) which can be armed or disarmed. The region shall be defined by a Polygon structure. Although this rule is defined within the context of the analytics service, it is intended to allow configuration of hardware motion detection as opposed to motion detection from scene description data. For a description of the parameters see <xref linkend="_Ref522789617" />.</para>
+      <para>The region motion detector detects any motion against the specified motion region. The rule is configured for an area (region of interest on the image source) which can be armed or disarmed. The region shall be defined by a Polygon structure. "VideoSource" in the Source of Messages refer to VideoSourceConfiguration token. Although this rule is defined within the context of the analytics service, it is intended to allow configuration of hardware motion detection as opposed to motion detection from scene description data. For a description of the parameters see <xref linkend="_Ref522789617" />.</para>
```

Comment:
Request to revert this change for the same reason mentioned above.

Reply:
No, it **must** be video source configuration. See definition of Polygon parameter where Bounds element is referred to that exist in VideoSourceConfiguration.